index.html

```diff
@@ -504,11 +504,12 @@ <h3>Use Cases and Requirements</h3>
           </li>
           <li>
 The processes of cryptographically protecting and verifying
-<a>verifiable credentials</a> and <a>verifiable presentations</a>, have to be
-deterministic, bi-directional, and lossless. Any verification of a
+<a>verifiable credentials</a> and <a>verifiable presentations</a> have to be
+deterministic, bi-directional, and lossless. Any
 <a>verifiable credential</a> or <a>verifiable presentation</a> has to be
-transformable to the generic data model defined in this document in a
-deterministic process so that it can be processed in an interoperable fashion.
+transformable to the JSON-LD data model defined in this document in a
```

Comment:
or `through`, or `by means of`, or similar.
```suggestion
transformable to the JSON-LD data model defined in this document via a
```